stopcovid/dialog/models/state.py

```diff
@@ -31,13 +31,12 @@ class UserProfile(pydantic.BaseModel):
     esl_opt_in: Optional[str] = None
     team_size: Optional[str] = None
 
-    def __str__(self):
+    def __str__(self) -> str:
         return f"lang={self.language}, validated={self.validated}, " f"name={self.name}"
 
     @pydantic.validator("language", pre=True, always=True)
-    def set_language(cls, value):
-        if value is not None:
-            return value.lower()[:2]
+    def set_language(cls, value: Optional[str]) -> Optional[str]:
+        return value.lower()[:2] if value else None
```

Comment:
we're changing the behavior for empty strings here.... is that going to be an issue?

Reply:
good call. No reason to have functionality changes in this PR. Will fix.